packages/broadcast/component.js

```diff
@@ -14,7 +14,7 @@ export class FabricBroadcast extends FabricWebComponent {
     const existing = this.shadowRoot.querySelector('#broadcast-toast');
 
     // Generate url
-    const dev = Boolean(this.getAttribute('dev'));
+    const dev = this.getAttribute('dev') != null;
```

Comment:
According to https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes boolean attributes should not have values "true" or "false". I think this should work in line with the spec then.